energyplus-version-updates.md

```diff
@@ -0,0 +1,115 @@
+# Review this file when updating EnergyPlus versions
```

Comment:
What is this file?

Reply:
This is a file that lists all object types used in this project, along with the files that they are found in. This is helpful when diff'ing IDDs between old & new EnergyPlus versions, since when you find an object with IDD diffs you can then check this file to know if you need to update this project accordingly (and what file to update).

Reply:
I'll go ahead and delete this file from the repo, since it may be confusing for others.